syntax = "proto2";

package ex.proto;

message RoleAttributes{
    //node
    optional string host_ip = 1;
    optional uint64 process_id = 2;
    optional string node_name = 3;
    optional uint64 node_id = 4;
    //channal
    optional string channel_name = 5;
    optional uint64 channel_id = 6;
    optional uint64 id = 7;//if of receiver or transmitter

}
